pkg/validators/test_am0004.go

```diff
@@ -0,0 +1,58 @@
+package validators
+
+import (
+	"github.com/mt-sre/addon-metadata-operator/api/v1alpha1"
+	"github.com/mt-sre/addon-metadata-operator/pkg/utils"
+)
+
+func init() {
+	TestRegistry.Add(TestAM0004{})
+}
+
+type TestAM0004 struct{}
+
+func (val TestAM0004) Name() string {
+	return AM0004.Name
+}
+
+func (val TestAM0004) Run(mb utils.MetaBundle) (bool, string, error) {
+	return AM0004.Runner(mb)
+}
+
+func (val TestAM0004) SucceedingCandidates() []utils.MetaBundle {
+	return []utils.MetaBundle{
+		{
+			AddonMeta: &v1alpha1.AddonMetadataSpec{
+				ID:              "random-operator",
+				TargetNamespace: "redhat-random-operator",
+				Namespaces: []string{
+					"redhat-random-operator",
+				},
+			},
+		},
```

Comment:
can we attach `testutils.DefaultSucceedingCandidates()` as well to the returning slice?

Ref: https://github.com/mt-sre/addon-metadata-operator/blob/57f68f5398eef130f5416d4719eb9106f296da0f/pkg/validators/test_am0006.go#L25-L49